monai/fl/client/monai_algo.py

```diff
@@ -357,15 +347,16 @@ def __init__(
         local_epochs: int = 1,
         send_weight_diff: bool = True,
         config_train_filename: str | list | None = "configs/train.json",
+        train_args: dict | None = None,
```

Comment:
Should this be additional_train_args and move it to the end of the argument list?

Reply:
Hi @wyli ,

I tried to be consistent with existing MONAI args, like:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py#L1154.
@holgerroth What do you think?

Thanks.

Reply:
Updated to "train_kwargs" now.

Thanks.